src/formatDate.js

```diff
@@ -50,7 +50,29 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateNew = date.split(fromFormat[3]);
+  const datePart = {};
+  const dateFormatted = [];
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    datePart[fromFormat[i]] = dateNew[i];
+  }
+
+  if (datePart.YYYY) {
+    datePart.YY = datePart.YYYY.slice(2);
+  }
+
+  if (datePart.YY >= 30) {
+    datePart.YYYY = '19' + datePart.YY;
```

Comment:
You can rewrite like this
```suggestion
    datePart.YYYY = `19${datePart.YY}`;
```